lib/samples/authenticate_with_oauth/authenticate_with_oauth.dart

```diff
@@ -23,40 +23,34 @@ class AuthenticateWithOAuth extends StatefulWidget {
   const AuthenticateWithOAuth({super.key});
 
   @override
-  State<AuthenticateWithOAuth> createState() =>
-      _AuthenticateWithOAuthState();
+  State<AuthenticateWithOAuth> createState() => _AuthenticateWithOAuthState();
 }
 
-class _AuthenticateWithOAuthState
-    extends State<AuthenticateWithOAuth>
+class _AuthenticateWithOAuthState extends State<AuthenticateWithOAuth>
     with SampleStateSupport
     implements ArcGISAuthenticationChallengeHandler {
+  // Create a controller for the map view.
+  final _mapViewController = ArcGISMapView.createController();
+
+  // Create an OAuthUserConfiguration.
   // This document describes the steps to configure OAuth for your app:
-  // https://developers.arcgis.com/documentation/mapping-apis-and-services/security/user-authentication/serverless-native-flow/
+  // https://developers.arcgis.com/documentation/security-and-authentication/user-authentication/flows/authorization-code-with-pkce/
   final _oauthUserConfiguration = OAuthUserConfiguration(
     portalUri: Uri.parse('https://www.arcgis.com'),
     clientId: 'T0A3SudETrIQndd2',
     redirectUri: Uri.parse('my-ags-flutter-app://auth'),
   );
-  final _mapViewController = ArcGISMapView.createController();
 
   @override
   void initState() {
     super.initState();
 
+    // Set this class to the arcGISAuthenticationChallengeHandler property on the authentication manager.
     // This class implements the ArcGISAuthenticationChallengeHandler interface,
     // which allows it to handle authentication challenges via calls to its
     // handleArcGISAuthenticationChallenge() method.
```

Comment:
This comment block also also appears over in `show_portal_uesr_info.dart` and prolly should get the same update. (I kinda cut-and-pasted some bits.)